### ISSUE TITLE
Update add domain to existing site to register (and cancel) .live domain

### ISSUE THREAD

**Issue by alisterscott** [Automattic/wp-e2e-tests#830]
We can use the sandbox method for adding a fake .live domain to an existing site and purchase it then refund and cancel it.

**Comment by alisterscott** [comment]
One thing we need to consider in adding a domain to a common account is the cart is shared across sessions, so concurrent tests running adding domains to same user share the cart and will potentially pay for not only their own domains - this could get really messy

**Comment by Stojdza** [comment]
@alisterscott We can do this with a non-default user, to make one for this case?

**Comment by alisterscott** [comment]
> We can do this with a non-default user, to make one for this case?

I think that would be best - I don't think there's a limit of how many domains we can register against an existing site - we'll soon see anyway.

We need to make sure sandbox mode is set for purchasing the .live domains

**Comment by Stojdza** [comment]
@alisterscott what do you think about this scenario:

0. We have existing user and existing _e2eflowtestinglivedomain.wordpress.com_ domain
1. Choose 'Add' domain
2. Search and select _.live_ domain (we should search something like e2eflowtestinglivedomain+timestamp, to be sure that every time domain is unique)
3. Purchase domain
4. Refund and cancel

Is `wp-manage-domains-spec.js` right place to put this test in?

**Comment by alisterscott** [comment]
> what do you think about this scenario:

sounds good

> Is wp-manage-domains-spec.js right place to put this test in?

Yes

Thank you

**Comment by Stojdza** [comment]
@alisterscott Do you maybe know, is `sandboxCookieValue` somehow related to the specific user?

When I'm using it with the user who is dedicated for this test I cannot open _Purchases_ page - it loads endlessly. When I try that with our default user, _Purchases_ page loads after some time.

**Comment by alisterscott** [comment]
> Do you maybe know, is sandboxCookieValue somehow related to the specific user?

I don't think so, but setting the cookie will use the store sandbox which means only sandboxed purchases will display, maybe there is an issue there with some of the sandbox store purchases
